templates/census-thank-you.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'layouts/_base.html' %}
+{% from 'components/panel/_macro.njk' import onsPanel %}
+
+{% block main %}
+
+  {% call onsPanel({
+    "type": "success",
+    "classes": "u-mb-s"
+    })
+  %}
+    <h1 data-qa="title"><span class="icon--check-green"></span>{{ _("Thank you for completing your census") }}</h1>
+
+      {% if content.census_type %}
+        {% if content.census_type =="IR" %}
```

Comment:
This logic could be handled in the context handler, and just have the string passed in to the template

Reply:
Problem is I would need to have the <strong> in the backend. I had a quick chat with Andrew and we decided that the lesser of 2 evils was the logic in the HTML